app/scripts/modules/core/src/managed/Environments.tsx

```diff
@@ -30,6 +31,12 @@ export function Environments(props: IEnvironmentsProps) {
 
   const totalContentWidth = isFiltersOpen ? CONTENT_WIDTH + 248 + 'px' : CONTENT_WIDTH + 'px';
 
+  const selectedArtifactDetails =
+    selectedArtifact &&
+    environments.artifacts
+      .find(({ name }) => name === selectedArtifact.name)
+      ?.versions.find(({ version }) => version === selectedArtifact.version);
```

Comment:
probably slow and bad to do this on every render, but going to likely change soon anyway.